app/services/sufia/workflow/workflow_by_admin_set_strategy.rb

```diff
@@ -0,0 +1,14 @@
+module Sufia
+  module Workflow
+    class WorkflowByAdminSetStrategy
+      def initialize(_work, attributes)
+        @admin_set_id = attributes.fetch(:admin_set_id, 'default')
```

Comment:
One more thing we should be aware of.  `attributes[:admin_set_id]` could be an empty string.  In that case we'll want to trap for it:

```ruby
@admin_set_id = attributes[:admin_set_id] if attributes[:admin_set_id].present?
```